skins/laika/src/components/pages/membership_form/Address.vue

```diff
@@ -125,11 +125,11 @@ export default Vue.extend( {
 	},
 	mounted() {
 		const initialFormValues = this.$props.initialFormValues;
-		if ( initialFormValues !== undefined ) {
-			Object.keys( this.$data.formData ).forEach( ( fieldName: string ) => {
+		Object.keys( this.$data.formData ).forEach( ( fieldName: string ) => {
+			if ( initialFormValues[ fieldName ] !== undefined ) {
```

Comment:
This was not in the ticket but previously, if no form data was supplied, all form fields were overwritten with "undefined" which caused the default country value to be empty instead of 'DE'. Maybe double-check that this is the best approach here.

Reply:
This is solved in #1561